dockerfile-configs/common-components.yaml

```
@@ -63,6 +63,14 @@
   - name: kubetail
     from: https://raw.githubusercontent.com/johanhaleby/kubetail/master/kubetail
     info: Bash script that enables you to aggregate (tail/follow) logs from multiple pods into one stream
+  - name: nerdctl
+    version: v0.9.0
+    from: https://github.com/containerd/nerdctl/releases/download/{version}/nerdctl-full-0.9.0-linux-amd64.tar.gz
+    to: /nerdctl-full-0.9.0-linux-amd64.tar.gz
+    command: |
+      tar Cxzvvf /usr/local nerdctl-full-0.9.0-linux-amd64.tar.gz &&\
+      rm -rf nerdctl-full-0.9.0-linux-amd64.tar.gz
```

```suggestion
      rm -f nerdctl-full-0.9.0-linux-amd64.tar.gz
```
recursive flag not needed in this case

---

is it possible to replace all the other hardcoded version occurrences with the version variable?

---

@petersutter it's doable however it requires some relative more work to change sth e.g. add a 'short_version' field in https://github.com/gardener/ops-toolbelt/blob/master/generator/lib/components.py#L93 and change other code which parse curl command , i haven't look into deep yet.
If you consider it's worth doing i will create another ticket to implement this.

meanwhile i see some similar usage e.g. https://github.com/gardener/ops-toolbelt/blob/master/dockerfile-configs/common-components.yaml#L55

what do you think?

---

ok this is probably too much effort, at least out of scope for this PR. Please ignore my previous comment

---

thanks @petersutter